backend/src/db/migrations/20200206190233-swap_latitude_with_longitude.js

```diff
@@ -0,0 +1,42 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description = `
+This migration swaps the value stored in Location.lat with the value
+of Location.lng. This is necessary as the values of lat and lng were
+stored incorrectly. For example Hamburg, Germany, was stored with the
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-11T11:52:46Z" title="Tuesday, February 11th 2020, 12:52:46 pm +01:00">Feb 11, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,55 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description =
+  'This migration swaps the value stored in Location.lat with the value of Location.lng.'
+
+export async function up(next) {
```


```suggestion
const swap = (next) => {
```

Reply:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-12T13:59:04Z" title="Wednesday, February 12th 2020, 2:59:04 pm +01:00">Feb 12, 2020</time>_

---


Done